### PR TITLE
fix: config

### PR DESCRIPTION
**Pull request by dvilelaf**
## Proposed changes

* Remove default values for all_participants to force the service to fail when the env var is not provided
* Update config format to use latest recommendations

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

File: packages/valory/services/contribution/service.yaml [hunk 1]
```diff
@@ -17,23 +17,25 @@ models:
...
       share_tm_config_on_startup: ${USE_ACN:bool:false}
-      token_uri_base: ${TOKEN_URI_BASE:str:https://pfp.staging.autonolas.tech/}
+      token_uri_base: ${TOKEN_URI_BASE:str:"https://pfp.staging.autonolas.tech/"}
       consensus:
         max_participants: 4
       setup:
-        all_participants: ${ALL_PARTICIPANTS:list:'[["0x976EA74026E726554dB657fA54763abd0C3a0aa9","0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65","0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc","0x14dC79964da2C08b23698B3D3cc7Ca32193d9955"]]'}
+        all_participants: ${ALL_PARTICIPANTS:list:[]}
+        safe_contract_address:
+        - '0x0000000000000000000000000000000000000000'
```
Comment (dvilelaf) [inline packages/valory/services/contribution/service.yaml:38]: Mandatory, even if the service does not interact with a safe

File: packages/valory/agents/contribution/aea-config.yaml [hunk 2]
```diff
@@ -150,17 +150,16 @@ models:
       service_id: contribution_service
       service_registry_address: null
       setup:
-        all_participants: ${SKILL_CONTRIBUTION_ABCI_MODELS_PARAMS_ARGS_SETUP_ALL_PARTICIPANTS:list:'[["0x976EA74026E726554dB657fA54763abd0C3a0aa9","0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65","0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc","0x14dC79964da2C08b23698B3D3cc7Ca32193d9955"]]'}
-        safe_contract_address:
-        - '0x0000000000000000000000000000000000000000'
+        all_participants: ${list:[]}
```
Comment (dvilelaf) [inline packages/valory/agents/contribution/aea-config.yaml:153]: No default values now to make it fail if the env var is not provided

File: packages/valory/services/contribution/service.yaml [hunk 3]
```diff
@@ -17,23 +17,25 @@ models:
...
+      leaderboard_sheet_id: ${LEADERBOARD_SHEET_ID:str:"1m7jUYBoK4bFF0F2ZRnT60wUCAMWGMJ_ZfALsLfW5Dxc"}
       observation_interval: ${OBSERVATION_INTERVAL:int:300}
       share_tm_config_on_startup: ${USE_ACN:bool:false}
-      token_uri_base: ${TOKEN_URI_BASE:str:https://pfp.staging.autonolas.tech/}
+      token_uri_base: ${TOKEN_URI_BASE:str:"https://pfp.staging.autonolas.tech/"}
       consensus:
         max_participants: 4
       setup:
-        all_participants: ${ALL_PARTICIPANTS:list:'[["0x976EA74026E726554dB657fA54763abd0C3a0aa9","0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65","0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc","0x14dC79964da2C08b23698B3D3cc7Ca32193d9955"]]'}
+        all_participants: ${ALL_PARTICIPANTS:list:[]}
```
Comment (dvilelaf) [inline packages/valory/services/contribution/service.yaml:36]: No default values now to make it fail if the env var is not provided

File: packages/valory/skills/contribution_abci/skill.yaml [hunk 4]
```diff
@@ -121,13 +121,8 @@ models:
...
       service_registry_address: null
       setup:
-        all_participants:
-        - - '0x976EA74026E726554dB657fA54763abd0C3a0aa9'
-          - '0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65'
-          - '0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc'
-          - '0x14dC79964da2C08b23698B3D3cc7Ca32193d9955'
-        safe_contract_address:
-        - '0x0000000000000000000000000000000000000000'
+        all_participants: []
```
Comment (dvilelaf) [inline packages/valory/skills/contribution_abci/skill.yaml:124]: No default values now to make it fail if the env var is not provided

File: packages/valory/skills/dynamic_nft_abci/skill.yaml [hunk 5]
```diff
@@ -139,7 +139,9 @@ models:
       round_timeout_seconds: 30.0
       service_id: contribution_service
       service_registry_address: null
-      setup: {}
+      setup:
+        all_participants: []
+        safe_contract_address: []
```
Comment (dvilelaf) [inline packages/valory/skills/dynamic_nft_abci/skill.yaml:144]: Not sure if needed also here, but I followed the price oracle implementation

**Review by Adamantios — APPROVED** [review]

**Pull request closed without merging**